lib/Service/PlatformService.php

```diff
@@ -117,13 +116,17 @@ public function getPlatforms(): array {
 		foreach ($this->platforms as $wrapper) {
 			$class = $wrapper->getClass();
 			try {
-				$platform = OC::$server->query((string)$class);
-				if ($platform instanceof IFullTextSearchPlatform) {
-					$wrapper->setPlatform($platform);
-					$platforms[] = $wrapper;
+				$platform = OC::$server->get((string)$class);
+				if (!($platform instanceof IFullTextSearchPlatform)) {
+					$this->miscService->log($class . ' does not implement ' . IFullTextSearchPlatform::class);
```

Comment:
- please use LoggerInterface as we'll get rid of MiscService
- also \OCP\Server::get();

Reply:
Done & force-pushed :heavy_check_mark: